src/hooks/screen/useChatScreenHook.ts

```diff
@@ -246,17 +291,23 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
           replyMessageId: null
         } as any);
       }
-      queue.addPriorityJob({
-        operationLabel: DatabaseOperationLabel.ChatScreen_UpdateChatSentStatus,
-        id: `${selectedChannel?.id}-${randomId}`,
-        priority: QueueJobPriority.HIGH,
-        forceAddToQueue: true,
-        task: async () => {
-          await currentChatSchema?.updateChatSentStatus(localDb, response);
-          initChatData();
-          refresh('channelList');
-        }
-      });
+      if (Platform.OS === 'android') {
+        await currentChatSchema?.updateChatSentStatus(localDb, response);
+        initChatData();
+        refresh('channelList');
+      } else {
+        queue.addPriorityJob({
+          operationLabel: DatabaseOperationLabel.ChatScreen_UpdateChatSentStatus,
+          id: `${selectedChannel?.id}-${randomId}`,
+          priority: QueueJobPriority.HIGH,
+          forceAddToQueue: true,
+          task: async () => {
+            await currentChatSchema?.updateChatSentStatus(localDb, response);
+            initChatData();
+            refresh('channelList');
+          }
+        });
+      }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change introduces platform-specific logic for updating the chat sent status. The same considerations apply here as well: ensure that these operations can be executed independently without causing race conditions or inconsistencies in the application state across different platforms.
```diff
+      if (Platform.OS === 'android') {
+        await currentChatSchema?.updateChatSentStatus(localDb, response);
+        initChatData();
+        refresh('channelList');
+      } else {
+        queue.addPriorityJob({
+          operationLabel: DatabaseOperationLabel.ChatScreen_UpdateChatSentStatus,
+          id: `${selectedChannel?.id}-${randomId}`,
+          priority: QueueJobPriority.HIGH,
+          forceAddToQueue: true,
+          task: async () => {
+            await currentChatSchema?.updateChatSentStatus(localDb, response);
+            initChatData();
+            refresh('channelList');
+          }
+        });
+      }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->